Tests/Rules/AvoidUnInitializedVarsInNewRunspaces.tests.ps1

```diff
@@ -0,0 +1,90 @@
+$directory = Split-Path -Parent $MyInvocation.MyCommand.Path
+$testRootDirectory = Split-Path -Parent $directory
+
+Import-Module (Join-Path $testRootDirectory "PSScriptAnalyzerTestHelper.psm1")

```

Comment:
I don't think we use the test helper (or do we?) and therefore I think we can remove the first 4 lines.

Reply:
Good point, we don't, so that can indeed be removed.